gas/testsuite/gas/riscv/insn-dsp.d

```diff
@@ -0,0 +1,245 @@
+#as: -mext-dsp
```

Comment:
This looks weird, I don't see any implementation about the option in this PR.

Reply:
I updated all test cases in the latest commit. Thanks for spotting it out!